apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -211,7 +222,7 @@ function CustomEventDialogs(props: CustomEventDialogProps) {
                     />
                 </DialogContent>
 
-                <DialogActions>
+                <DialogActions style={{ padding: '0px 16px 16px' }}>
```

Comment:
question: why are we adding padding to the footer/dialog actions? It's not necessarily incorrect, but I think we would prefer to not add arbitrary styles in favor of maintaining styles across components (i.e. other dialogs). 

To be clear, you can (and should) argue for the styling if you have a compelling reason!